src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -2892,16 +2892,28 @@ public J visitProperty(KtProperty property, ExecutionContext data) {
         }
 
         if (getter != null || setter != null || receiver != null || typeConstraints != null) {
+            List<JRightPadded<J.MethodDeclaration>> rps = new ArrayList<>(2);
+
+            if (setter != null) {
+                rps.add(padRight(setter, Space.EMPTY));
```

Comment:
This looks a bit strange. Does the right padding represent the space before an optional semi-colon? Looks like `maybeTrailingSemicolon()` should be used here.

Reply:
good catch, The trailing semi-colon of setter/getter is not handled by us. I added a test case for this and fixed.